Cargo.toml

```diff
@@ -115,58 +116,11 @@ numa = [
 #
 # This list is ordered alphabetically.
 [profile.dev.package]
-bitvec = { opt-level = 3 }
```

Comment:
These changes should not be committed

Reply:
Fixed in commit@https://github.com/subspace/space-acres/commit/bd5fbd77b7ebd8b8073c93562d2ac8437f11f5ee